back/engines/commercial/project_folders/app/models/project_folders/patches/project.rb

```diff
@@ -16,6 +16,10 @@ def folder
         admin_publication&.parent&.publication
       end
 
+      def folder_id
+        admin_publication&.parent&.publication_id
+      end
```

Comment:
Wasn't this here before? 🤔

Reply:
No, but there was a `folder` method already :)